extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -234,7 +265,7 @@ object PrivilegesBuilder {
         outputObjs += tablePrivileges(table)
 
       case "AlterViewAsCommand" =>
-        val view = getPlanField[TableIdentifier]("name")
+        val view = getPlanField[TableIdentifier]("name", true)
```

Comment:
we can remain `val view = getPlanField[TableIdentifier]("name")`, and add another  `private def setCurrentDBIfNecessary` method to set currentDb.

Reply:
I got it.